openapi/bitly/.gitignore

```diff
@@ -0,0 +1,2 @@
+target
+tests
```

Comment:
You can remove this line

Reply:
Hi, Did the suggested changes. 
/Thanks